napari_tiff/napari_tiff_metadata.py

```diff
@@ -0,0 +1,395 @@
+    )
+    return kwargs
+
+
+def get_ome_tiff_metadata(tif: TiffFile) -> dict[str, Any]:
+    ome_metadata = xml2dict(tif.ome_metadata).get("OME")
+    image_metadata = ensure_list(ome_metadata.get("Image", {}))[0]
+    pixels = image_metadata.get("Pixels", {})
+
+    pixel_size = []
+    size = float(pixels.get("PhysicalSizeX", 0))
+    if size > 0:
+        pixel_size.append(
+            get_value_units_micrometer(size, pixels.get("PhysicalSizeXUnit"))
+        )
+    size = float(pixels.get("PhysicalSizeY", 0))
+    if size > 0:
+        pixel_size.append(
+            get_value_units_micrometer(size, pixels.get("PhysicalSizeYUnit"))
+        )
+
+    series = tif.series[0]
+    shape = series.shape
+    dtype = series.dtype
+    axes = series.axes.lower().replace("s", "c")
+    if "c" in axes:
+        channel_axis = axes.index("c")
+        nchannels = shape[channel_axis]
+    else:
+        channel_axis = None
+        nchannels = 1
+
+    channels = ensure_list(pixels.get("Channel", []))
+    if len(channels) > nchannels:
+        nchannels = len(channels)
+
+    is_rgb = series.keyframe.photometric == PHOTOMETRIC.RGB and nchannels in (3, 4)
+
+    if is_rgb:
+        # channels_axis appears to be incompatible with RGB channels
+        channel_axis = None
+
+    names = []
+    contrast_limits = []
+    colormaps = []
+    blendings = []
+    visibles = []
+
+    scale = None
+    if pixel_size:
+        scale = pixel_size
+
+    for channeli, channel in enumerate(channels):
+        name = channel.get("Name")
+        color = channel.get("Color")
+        colormap = None
+        if color:
+            colormap = int_to_rgba(int(color))
+        elif is_rgb and len(channels) > 1:
+            # separate channels provided for RGB (with missing color)
+            colormap = ["red", "green", "blue", "alpha"][channeli]
+            if not name:
+                name = colormap
+
+        blending = "additive"
+        visible = True
+
+        if dtype.kind == "f":
+            contrast_limit = None
+        else:
+            info = numpy.iinfo(dtype)
+            contrast_limit = (info.min, info.max)
+
+        if len(channels) > 1:
+            names.append(name)
+            blendings.append(blending)
+            contrast_limits.append(contrast_limit)
+            colormaps.append(colormap)
+            visibles.append(visible)
+        else:
+            names = name
+            blendings = blending
+            contrast_limits = contrast_limit
+            colormaps = colormap
+            visibles = visible
+
+    kwargs = dict(
+        rgb=is_rgb,
+        channel_axis=channel_axis,
+        name=names,
+        scale=scale,
+        colormap=colormaps,
+        contrast_limits=contrast_limits,
+        blending=blendings,
+        visible=visibles,
+    )
+    return kwargs
+
+
+def get_value_units_micrometer(value: float, unit: str = None) -> float:
+    unit_conversions = {
+        "nm": 1e-3,
+        "µm": 1,
```

Comment:
I do not remember why, but I have such a version of `µm` in my codebase. I cannot find an example file fast, but I think that add it should be painless.

```suggestion
        "µm": 1,
        "\\u00B5m": 1,
```

Reply:
Ok, but we'll need to add a comment indicating that this is the unicode "micro sign" character

Reply:
Done.